docs/sources/flow/setup/start-agent.md

```diff
@@ -146,26 +145,26 @@ If you downloaded the standalone binary, you must run the agent from a terminal
 To start Grafana Agent on Linux, macOS, or FreeBSD, run the following command in a terminal window:
 
 ```shell
-AGENT_MODE=flow BINARY_PATH run CONFIG_FILE
+AGENT_MODE=flow BINARY run CONFIG_FILE
 ```
 
 Replace the following:
 
-* `BINARY_PATH`: The path to the extracted binary.
-* `CONFIG_FILE`: The path to a Grafana Agent configuration file.
+* `BINARY`: The Grafana Agent binary file. Include the path to the binary file.
+* `CONFIG_FILE`: The Grafana Agent configuration file. Include the path to the configuration file.
```

Comment:
Hm, maybe I'm not sure I'm following what "Include the path to the configuration file" means. What does a value of CONFIG_FILE mean which "includes the path?"

Reply:
I've reverted this change in the next commit.